### DISCUSSION TITLE
Volt's title method doesn't seem to work in full page component

### DISCUSSION THREAD

**Discussion by MUHAMMADSiyab** [livewire/livewire#8150] [1. Help] [answered]
**Laravel version: ^10.10**
**Livewire version: ^3.4**
**Volt version: ^1.0**

Using Full Page Volt component doesn't apply title to the page using `title` method.

**blog-posts.blade.php**:

```
<?php

use App\Models\Post;
use Livewire\WithPagination;

use function Livewire\Volt\{state, layout, computed, uses, title};

uses([WithPagination::class]);

layout('components.layouts.home');
title('Blog');


$posts = computed(function () {
    return Post::latest()->simplePaginate(10);
});

?>

@foreach ($this->posts as $post)
    <article id="blog-post" class="mb-8 border-b p-2 pb-6 ">
        <header>
            <h3 class="text-xl md:text-3xl font-semibold"> {{ $post->title }}</h3>
        </header>
    </article>
 @endforeach
```

Layout contains title like this:

`<title>{{ config('app.name', 'Laravel') }}</title>`

The page still shows the default `Laravel` title

**Comment by danie-ramdhani** [comment, marked as answer]
use `<title>{{ $title ?? config('app.name') }}</title>`

Livewire using discussions to track bugs and ideas so this is the bug report.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147
you can close the issue.

**Reply by MUHAMMADSiyab** [reply]
Thank you

**Comment by joshhanley** [comment]
@MUHAMMADSiyab please don’t open a discussion and issue future. This discussion is the bug report.

Also we don’t maintain Volt, so you should raise this on the Volt repo.
